internal/util/importutil/parquet_parser_test.go

```diff
@@ -584,39 +689,39 @@ func TestParquetReader(t *testing.T) {
 		})
 
 		t.Run("read not int8 field", func(t *testing.T) {
-			columnReader := parquetParser.columnMap["FieldInt16"]
+			columnReader := parquetParser.columnMap["FieldBool"]
```

Comment:
Retain the previous data type.
 Employing the former data type should result in an error.